apps/ledger-live-desktop/src/renderer/IPCTransport.ts

```diff
@@ -87,18 +88,18 @@ export class IPCTransport extends Transport {
 
   async exchange(apdu: Buffer): Promise<Buffer> {
     const apduHex = apdu.toString("hex");
-    log("apdu", "=> " + apduHex);
+    log("ipc-apdu", "=> " + apduHex);
     const responseHex = await rendererRequest(transportExchangeChannel, {
       descriptor: this.id,
       apduHex,
     });
-    log("apdu", "<= " + responseHex);
+    log("ipc-apdu", "<= " + responseHex);
```

Comment:
i remember the logsviewer have a specific code have an "extract apdus" button that will recreate a apdu.txt on the logs of type "apdu", but i imagine they were now duplicated and so you had to do the ipc-adpu one?

Reply:
Exactly, we would get `apdu` from the IPCTransport and `apdu` from the actual `Transport`